canonical-web-and-design/ubuntu.com
# Comment out Latest release from megamenu

## Done

- Comment out Latest release from megamenu

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/alternative-downloads#download
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that there is no latest white button, the green ones might not be 20.04 yet


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2622
